config/action.d/cloudflare.conf

```diff
@@ -43,13 +48,18 @@ actionban = curl https://www.cloudflare.com/api_json.html -d 'a=ban' -d 'tkn=<cf
 #          <time>  unix timestamp of the ban time
 # Values:  CMD
 #
-actionunban = curl https://www.cloudflare.com/api_json.html -d 'a=nul' -d 'tkn=<cftoken>' -d 'email=<cfuser>' -d 'key=<ip>'
-
+actionunban = curl -s -o /dev/null https://www.cloudflare.com/api_json.html -d 'a=nul' -d 'tkn=<cftoken>' -d 'email=<cfuser>' -d 'key=<ip>'
 
 [Init]
 
-# Default Cloudflare API token 
-cftoken = 
+# This action needs to be set up in jail.local or in jail.d/somefile.conf
+#
+# action = %(action_cf_mwl)s
```

Comment:
Here it is. Down below:
https://github.com/fail2ban/fail2ban/pull/1085/files#diff-04e245e7ddfd46056df5e124cd8d5c11R179